### DISCUSSION TITLE
Stock take and Import/export receipt

### DISCUSSION THREAD

**Discussion by minhkhoinguyen** [Blair2004/NexoPOS#268] [Ideas]
Stock take: inventory check to manage goods periodically.
Every time you postimport/export goods, you will need forms to control
![Stock Take](https://user-images.githubusercontent.com/12565814/129581627-851f2113-879f-437d-82bc-1b6a7d34b29e.png)
 
![Import   export receipt](https://user-images.githubusercontent.com/12565814/129581616-e5a12770-3ea3-4bd9-a78f-386ef238f648.png)

**Comment by Blair2004** [comment]
Hi, 
How this can be useful and where ideally this should be implemented.
Thanks
